src/platform/site-wide/mega-menu/containers/Main.jsx

```diff
@@ -185,14 +183,10 @@ const mapStateToProps = (state, ownProps) => {
   const defaultLinks = ownProps?.megaMenuData ? [...ownProps.megaMenuData] : [];
 
   if (loggedIn) {
-    defaultLinks.push(MY_VA_LINK);
+    defaultLinks.push(MY_VA_LINK, MY_HEALTH_LINK);
   }
 
-  const authenticatedLinks = toggleValues(state)[
-    FEATURE_FLAG_NAMES.mhvLandingPageEnabled
-  ]
-    ? [{ ...MY_HEALTH_LINK }]
-    : undefined;
+  const authenticatedLinks = [];
```

Comment:
Why not removed this too?

Reply:
I could go ham on deleting code in the mega-menu, but then it would be outside the scope of removing this feature toggle.